grass7/imagery/i.sentinel/i.sentinel.download/i.sentinel.download.py

```diff
@@ -101,6 +101,13 @@
 #% guisection: Filter
 #%end
 #%option
+#% key: relativeorbitnumber
+#% type: integer
+#% multiple: no
+#% description: Relative orbit number to download (Sen-1: from 1 to 175; Sen-2: from 1 to 143)
```

Comment:
```suggestion
#% description: Relative orbit number to download (Sentinel-1: from 1 to 175; Sentinel-2: from 1 to 143)
```

(just to avoid abbreviations for nowcomers)